pkgs/tools/package-management/nix/default.nix

```diff
@@ -155,7 +155,7 @@ in rec {
     name = "nix-1.11.16";
     src = fetchurl {
       url = "http://nixos.org/releases/nix/${name}/${name}.tar.xz";
-      sha256 = "7024d327314bf92c1d3e6cccd944929828a44b24093954036bfb0115a92f5a14";
+      sha256 = "0ca5782fc37d62238d13a620a7b4bff6a200bab1bd63003709249a776162357c";
     };
   }) // { perl-bindings = nixStable; };
```

Comment:
Not to nag, but just checking: is this intended to use perl bindings from nixStable (nix 2)?